tests/flatpages_tests/test_models.py

```diff
@@ -13,3 +13,7 @@ def test_get_absolute_url_urlencodes(self):
     def test_get_absolute_url_honors_script_prefix(self):
         pf = FlatPage(title="Tea!", url='/tea/')
         self.assertEqual(pf.get_absolute_url(), '/beverages/tea/')
+
+    def test_str(self):
+        pf = FlatPage(title="Café!", url='/café/')
```

Comment:
Use single quotes for the Cafe! string -- perhaps a commit that precedes that additional test could refactor tests to use:
```
def setUp(self):
    self.page = FlatPage(title='Café!', url='/café/')
```